saml/identityProviders/okta/file/oktaSession.go

```diff
@@ -37,6 +37,16 @@ const (
 
 type OktaSessionStorage struct{}
 
+func (o *OktaSessionStorage) ClearSessions() {
+	sessions := make([]OktaSessionCache, 0)
+	sessionsJSON, _ := json.Marshal(sessions)
+	bmxHome := path.Join(userHomeDir(), ".bmx")
+	if _, err := os.Stat(bmxHome); os.IsNotExist(err) {
+		os.MkdirAll(bmxHome, os.ModeDir|os.ModePerm)
+	}
+	ioutil.WriteFile(path.Join(userHomeDir(), ".bmx", sessionFileName), sessionsJSON, 0644)
+}
```

Comment:
Minor nit, this logic is duplicated from `SaveSessions` and should perhaps be abstracted into another function that accepts a list of sessions to save.

Reply:
I'll make a follow-up PR for this file, as there are some other things that could be moved to their own files (e.g. `userHomeDir+.bmx+sessionFileName`).